api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -128,26 +128,28 @@
   private static final String BUCKET_NAME = "workspace-bucket";
 
   private static final Concept CLIENT_CONCEPT_1 = new Concept()
-      .conceptId(123L)
-      .conceptName("a concept")
-      .standardConcept(true)
-      .conceptCode("conceptA")
-      .conceptClassId("classId")
-      .vocabularyId("V1")
-      .domainId("Condition")
-      .countValue(123L)
-      .prevalence(0.2F);
+          .conceptId(123L)
+          .conceptName("a concept")
+          .standardConcept(true)
```

Comment:
Curious why this isn't an "S" or "C" or null ?

Reply:
True is converted to S and false to C and null is left as it is in the make concept below.